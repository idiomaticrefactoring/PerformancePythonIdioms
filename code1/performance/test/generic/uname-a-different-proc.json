{"kernel_name":"Linux","node_name":"mymachine","kernel_release":"2.6.18-194.e15PAE","operating_system":"GNU/Linux","processor":"i386","hardware_platform":"i686","machine":"i686","kernel_version":"#1 SMP Fri Apr 2 15:37:44 EDT 2010"}
