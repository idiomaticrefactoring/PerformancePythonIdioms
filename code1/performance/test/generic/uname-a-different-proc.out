Linux mymachine 2.6.18-194.e15PAE #1 SMP Fri Apr 2 15:37:44 EDT 2010 i686 i686 i386 GNU/Linux
